package.json

```diff
@@ -37,19 +37,17 @@
   },
   "dependencies": {
     "@actions/core": "^1.10.1",
-    "@actions/exec": "^1.1.1",
     "@actions/github": "^6.0.0",
-    "@octokit/rest": "^20.0.2",
+    "@octokit/request-error": "^5.0.1",
```

Comment:
Unlisted dep

Reply:
This is already defined as a transitive dependency for multiple other dependencies in [package-lock.json,](https://github.com/github/dependency-submission-toolkit/blob/5361b6878ba6de3b2a22890462e4a2e7cc08f7db/package-lock.json#L754) why do we need to be explicit here?

Reply:
Having transitive dependencies always makes the project fragile, so I believe it makes sense to declare them as direct dependencies 🤔

Reply:
With Dependabot recently [supporting transitive dependencies](https://github.blog/changelog/2022-09-07-dependabot-unlocks-transitive-dependencies-for-npm-projects/) I don't think this is a problem, but we need to [enable it for NPM](https://github.com/github/dependency-submission-toolkit/blob/main/.github/dependabot.yml)!

The PR was merged after 2.0.2 was cut, thank you again!

Reply:
I will check that! Thanks a lot!